e2e/volume_ops_test.go

```diff
@@ -112,27 +111,91 @@ func testVolumeCreate(t *testing.T) {
 	createReq.Name = "##@@#@!#@!!@#"
 	_, err = client.VolumeCreate(createReq)
 	r.NotNil(err)
+
```

Comment:
This test needs cleaning. It's grown so huge. You can have the flags test as a separate test. The code you write below doesn't depend on the earlier code in the test.

Reply:
done